XamlControlsGallery/ControlPages/AnimatedVisualPlayerPage.xaml.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
```

Comment:
Please remove unnecessary "USING"s.

Reply:
I used the Removed Unused Usings feature in VS2017 and these don't get removed.

Reply:
Removed